backend/src/main/java/gov/cdc/usds/simplereport/db/model/Result.java

```diff
@@ -0,0 +1,57 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import gov.cdc.usds.simplereport.api.Translators;
+import gov.cdc.usds.simplereport.db.model.auxiliary.TestResult;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Entity
+public class Result extends EternalAuditedEntity {
+
+  @ManyToOne(optional = false)
+  @JoinColumn(name = "test_event_id", nullable = false, updatable = false)
+  private TestEvent testEvent;
+
+  @ManyToOne(optional = false)
+  @JoinColumn(name = "test_order_id", nullable = false, updatable = false)
+  private TestOrder testOrder;
+
+  @ManyToOne(optional = false)
+  @JoinColumn(name = "disease_id", nullable = false, updatable = false)
+  private SupportedDisease disease;
+
+  // Stored as LOINC
+  @Column(nullable = false)
+  private String result;
```

Comment:
it might be worth renaming the field to resultLOINC to be more accurate, 
that way we can also change `getRawTestResult/setRawTestResult` to `getTestResultLoinc/setTestResultLoinc` and at that point the @Getter will take care of generating the getter with the correct name
we can easily rename the field by adding the column name to the `@Column` here without renaming the db column name if thats too much trouble